src/ontology/cl-edit.owl

```diff
@@ -3604,6 +3604,7 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000025 "MESH:D010063")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000025 "mature oocyte")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000025 "ovum")
 AnnotationAssertion(rdfs:label obo:CL_0000025 "egg cell")
+AnnotationAssertion(<http://xmlns.com/foaf/0.1/depicted_by> obo:CL_0000025 "https://www.swissbiopics.org/name/Animal_egg_cell")
```

Comment:
It's expected to use a link with an image extension and the datatype `xsd:anyURI`. You could embed a `seeAlso` annotation to add the link you used here.

I found the link to the static image by "copying link address" (right click) in the arrow icon next to the title "Animal Egg cell" https://www.swissbiopics.org/api/image/Egg_cell.svg

Reply:
Please take a look at the same comment for each of your changes in the PR.

Reply:
@anitacaron thanks! I saw the SVG link too; the original URL seems to be a bit more informative, because the image becomes interactive, and one can see the different parts of the cell type. 

I'll make the changes, but do you know if these expectations are documented anywhere? 

http://xmlns.com/foaf/0.1/depicted_by seems to go 404, while http://xmlns.com/foaf/0.1/depiction exists with some documentation: 

https://xmlns.com/foaf/0.1/#term_depiction

Reply:
I've changed the datatype, but I am a bit unsure with regards to the best approach for the URL. 

I imagine that people accessing the ontology on OLS, for example, would have a better user experience by being directed to the interactive image.

Maybe we can discuss that briefly in the next CL call?

Reply:
You're right. I can't find the property `foaf:depicted_by`, only `foaf:depiction`. We should change the property in another issue. It's used in many terms in Uberon, but only one here in CL.

If you checked the docs for `foaf:depiction`, the range is [foaf:Image](http://xmlns.com/foaf/0.1/#term_Image).

>Digital images (such as JPEG, PNG, GIF bitmaps, SVG diagrams etc.) are examples of [foaf:Image](http://xmlns.com/foaf/0.1/#term_Image).

OLS uses the property to render the image and only works with a digital image. [Example ](https://www.ebi.ac.uk/ols4/ontologies/uberon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0000401). It won't work if you keep the link to a web page.

Reply:
Feel free to add the link to the interactive image using the annotation `rdfs:seelAlso`